.gitignore

```diff
@@ -1,11 +1,8 @@
-build-vendor/
-vendor/
-composer.lock
-composer-compat.json
-composer-compat.lock
-manual/
-dist/
-__pycache__
-.php_cs.cache
-psysh.phar
-psysh-compat.phar
+/build/
```

Comment:
I added the `/` prefix, it's usually a better practice to ensure those rules applies only to this project.